clusters/app.ci/supplemental-ci-images/coreos/coreos-stream8-buildroot/build-base.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -xeuo pipefail
+# First, fix coreutils
+yum -y swap coreutils{-single,}
```

Comment:
nit: I think we can use `dnf` instead of `yum` here

Reply:
There's an alias thankfully. We  can move it to dnf/microdnf later

Reply:
> There's an alias thankfully. We can move it to dnf/microdnf later

I know it's an alias but all these files are new, we could just `search and replace` :smiley: 

(also, I labeled my comment as `nitpick` so, not a blocker!)